ProjectLighthouse.Servers.GameServer/Controllers/Resources/PhotosController.cs

```diff
@@ -83,7 +86,7 @@ public async Task<IActionResult> UploadPhoto()
                     break;
             }
 
-            if (validLevel) photo.SlotId = photo.XmlLevelInfo.SlotId;
+            if (validLevel) photo.SlotId = (adventureSlot != null) ? adventureSlot.SlotId : photo.XmlLevelInfo.SlotId;
```

Comment:
```suggestion
            if (validLevel) photo.SlotId = photo.XmlLevelInfo.SlotId;
```
This can be simplified because of the changes above